variables.tf

```
@@ -271,6 +284,12 @@ variable "prometheus_enabled" {
   type        = string
 }
 
+variable "metastore_uri" {
+  description = "Metastore URI."
```

```suggestion
  description = "URI of the metastore where tables to be cleaned-up are located. Required for Beekeeper Metadata Cleanup."
```

You can use `terraform-docs markdown .` command to generate your README section for all variables.